### DISCUSSION TITLE
CSS Framework

### DISCUSSION THREAD

**Discussion by raghusaripalli** [raghusaripalli/GatorGiveaway#28] [Ideas]
For Sprint 2 here are some good looking CSS material frameworks for building response UI:
- [Material UI for React](https://mui.com/getting-started/usage/)
- [Material Design](https://material.io/components?platform=web)
- [Materialize CSS](https://materializecss.com/)

Open to any other suggestions and free open-source templates built using these frameworks that suit our web app.

**Comment by ksmSujay95** [comment]
These frameworks looks good to implement, will let the team know if anything new and useful sources found.

**Comment by ypalnati** [comment]
Material framework looks good to implement . Found this resource online : https://mui.com/getting-started/installation/
Let me know your opinion on this.

**Comment by tejasri555** [comment]
Yeah looks good for me too. Let's proceed with this.
